libs/components/ag-grid/src/lib/styles/_modern-dark.scss

```diff
@@ -3,6 +3,10 @@
 
 .ag-theme-sky-modern-dark {
   ag-grid-angular:not(.sky-ag-grid-editable) {
+    .ag-row-hover::before {
+      background-color: transparent;
+    }
+
```

Comment:
angular materials adds a background color when hovering over a row but only in dark mode, is there a better way to remove that?